spec/spec_helper.rb

```diff
@@ -0,0 +1,100 @@
+# This file was generated by the `rspec --init` command. Conventionally, all
```

Comment:
To avoid giving you the solution directly, kindly google the error message and apply the solution found online, if you still need help, kindly message me on slack :slightly_smiling_face: